lms/services/lti_grading/factory.py

```diff
@@ -8,7 +8,10 @@ def service_factory(_context, request):
 
     if application_instance.lti_version == "1.3.0":
         return LTI13GradingService(
-            line_item_url=request.parsed_params.get("lis_outcome_service_url"),
+            # Pick the value from the right dictionary depending on the context we are running
+            # either an API call from the frontend (parsed_params) or inside an LTI launch (lti_params).
+            line_item_url=request.parsed_params.get("lis_outcome_service_url")
```

Comment:
This never been a problem before because we never used the grading service during a launch only one the read/record grade API calls.